policyengine_us/variables/household/expense/housing/housing_cost.py

```diff
@@ -8,4 +8,4 @@ class housing_cost(Variable):
     unit = USD
     definition_period = YEAR
 
-    adds = ["rent"]
+    adds = ["rent", "property_tax_primary_residence", "homeowners_association_fees", "mortgage_payments"]
```

Comment:
not real_estate_taxes?

Reply:
Property tax for the primary residence is a sum of real estate taxes:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/eef7cdf4-cb35-4233-b503-8c3f266258dc)